dotcom-rendering/src/web/components/MostViewedRight.tsx

```diff
@@ -47,8 +47,8 @@ export const MostViewedRight = ({
 
 	if (data) {
 		const trails: TrailType[] = data.trails
-			.map(decideTrail)
-			.slice(0, limitItems);
+			.slice(0, limitItems)
+			.map(decideTrail);
```

Comment:
Slice first, so you map on less!

Reply:
Switched this order because calling `slice` first is (potentially) more efficient for the subsequent `map`